lib/rucio/core/rule.py

```diff
@@ -2418,23 +2414,26 @@ def examine_rule(rule_id, *, session: "Session"):
 
 
 @transactional_session
-def get_evaluation_backlog(expiration_time=600, *, session: "Session"):
+def get_evaluation_backlog(expiration_time: int = 600, *, session: "Session") -> tuple[int, datetime]:
     """
     Counts the number of entries in the rule evaluation backlog.
     (Number of files to be evaluated)
 
     :returns:     Tuple (Count, Datetime of oldest entry)
     """
 
-    result = REGION.get('rule_evaluation_backlog', expiration_time=expiration_time)
-    if result is NO_VALUE:
-        result = session.query(func.count(models.UpdatedDID.created_at), func.min(models.UpdatedDID.created_at)).one()
+    cached_backlog = REGION.get('rule_evaluation_backlog', expiration_time=expiration_time)
+    if cached_backlog is NO_VALUE:
+        stmt = select(func.count(models.UpdatedDID.created_at), func.min(models.UpdatedDID.created_at))
+        result = session.execute(stmt).scalars().one()
         REGION.set('rule_evaluation_backlog', result)
-    return result
+        return result
+    else:
+        return cached_backlog  # type: ignore
```

Comment:
See https://github.com/rucio/rucio/pull/6493/files#r1521715742. Please note how @erlingstaff has formatted the query. I think you should copy his work and apply the corrections on top of it so that he can ignore this function completely.